DQMOffline/JetMET/src/JetAnalyzer_HeavyIons.cc

```diff
@@ -795,11 +795,15 @@ void JetAnalyzer_HeavyIons::analyze(const edm::Event& mEvent, const edm::EventSe
   edm::Handle<int> cbin;
   mEvent.getByToken(centralityBinToken, cbin);
   
-  if (!cent.isValid()) return;
+  if (!cent.isValid()){
+    edm::LogError("JetAnalyzer_HeavyIons") << "invalid collection: centrality " << std::endl;
```

Comment:
If this is a real problem, better throw an exception (or let the framework throw it).
Since a while now, it can take a long time until a LogError is noticed.